lib/main.dart

```diff
@@ -36,7 +63,45 @@ Future<void> main() async {
   );
 }
 
-class MyApp extends StatelessWidget {
+class MyApp extends StatefulWidget {
```

Comment:
Don't convert the MyApp to stateful, shift the token generation and initialization of push notification in dashboard.dart